src/api/index.ts

```diff
@@ -22,6 +23,27 @@ export const createApp = (config: Config): express.Express => {
     throw new Error();
   });
 
+  app.use(express.static('public'));
+  app.use(express.urlencoded({ extended: true }));
+
+  app.post('/', async (req: Request, res, next) => {
+    try {
+      const input: PrintInput = req.body;
+      if (req.accepts('application/pdf')) {
+        res.setHeader('content-type', 'application/pdf');
+        if (input.attachment) {
+          res.setHeader('content-disposition', `attachment; filename="${req.body.attachment}"`);
```

Comment:
You'll potentially have the same issue I had with seshat recently... If the filename I'm providing contains weird chars the header will be invalid and this won't fly well with chrome and other browsers.
It's probably best to `encodeURIComponent(req.body.attachment)`